src/main/java/org/jenkinsci/plugins/parallel_test_executor/TestCaseCentricFormat.java

```diff
@@ -0,0 +1,21 @@
+package org.jenkinsci.plugins.parallel_test_executor;
+
+public enum TestCaseCentricFormat  {
+    DISABLED ("Disabled"),
+    TCNAME ("Enabled - testCaseName"),
+    CLNAME_DOT_TCNAME ("Enabled - className.testCaseName");
+
+    private final String description;
+
+    TestCaseCentricFormat(String description) {
+        this.description = description;
+    }
+    
+    public String getDescription() {
+        return description;
+    }
+
+    static public boolean isEnabled(TestCaseCentricFormat format) {
+        return (format != null) && (format != TestCaseCentricFormat.DISABLED);

```

Comment:
Probably better to have the `format` always be passed around as non-null, and for this to be an instance method.